### ISSUE TITLE
`pad_end` in `column_text_to_width` should be `string_width.align`

### ISSUE THREAD

**Issue by yoshi-monster** [ollien/tobble#1]
You limit the width of the columns like [this](https://github.com/ollien/tobble/blob/main/src/tobble.gleam#L484):

```gleam
 text
  |> string_width.limit(
    to: string_width.Size(rows: string.length(text), columns: width),
    ellipsis: "",
  )
  |> string.pad_end(to: width, with: " ")
```

however, this can overflow the width when using wide characters. :purple_heart: for example is 2 cells wide; using this function to place it in a 5 wide table column would produce the string `💜    ` (a heart with 4 spaces) since :purple_heart: is counted as 1 character inside `pad_end`, resulting in a string that is 2+4=6 cells wide.

You can use [string_width.align](https://hexdocs.pm/string_width/string_width.html#align) instead!

```gleam
 text
  |> string_width.limit(
    to: string_width.Size(rows: string.length(text), columns: width),
    ellipsis: "",
  )
  |> string_width.align(align: string_width.Left, to: width, with: " ")
```

~yoshi :purple_heart:

**Comment by ollien** [comment]
Thanks for the issue @joshi-monster !

I had wrangled trying to work with wide characters, but assumed after a while it was hard due to terminal emulator weirdness. Seems I was wrong!

I think your suggestion has a slight bug; `string.pad_end("", to: 10, with)` returns ten spaces, while `string_width.align("", align: string_width.Left, to:  10, with: " ")` returns an empty string. I think I can just cover this case with a conditional, though.

**Comment by yoshi-monster** [comment]
Terminal emulator weirdness with wide characters is kinda the whole point of `string_width` :smile: But this shouldn't be happening, I'll change that tomorrow :)

**Comment by ollien** [comment]
Fixed in v1.0.1  :)
